packages/fxa-settings/src/components/GetDataTrio/index.tsx

```diff
@@ -30,12 +36,22 @@ const recoveryCodesPrintTemplate = (
   `;
 };
 
-export const GetDataTrio = ({ value, onAction }: GetDataTrioProps) => {
+export const GetDataTrio = ({
+  value,
+  contentType,
+  onAction,
+}: GetDataTrioProps) => {
   const { l10n } = useLocalization();
+
+  // Fall back to 'Firefox' just in case.
+  if (contentType == null) {
+    contentType = 'Firefox';
+  }
+
   const pageTitle = l10n.getString(
-    'get-data-trio-title',
+    'get-data-trio-title-' + contentType.toLowerCase().replace(/\s/g, '-'),
```

Comment:
Could use regex here or have a map containing the messageIds.

Reply:
I suppose a regex is being used in the replace call. That being said, I like can see the value of stating the l10n id explicitly. Probably better for maintenance / dev-ex when doing a text search. I'll switch it over to a map.